creation/interfaces-and-networks.md

```diff
@@ -670,17 +670,12 @@ device, using the
 [vfio](https://www.kernel.org/doc/Documentation/vfio.txt) userspace
 interface, to maintain high networking performance.
 
-    kind: VM
-    spec:
-      domain:
-        devices:
-          interfaces:
-            - name: sriov-net
-              sriov: {}
-      networks:
-      - name: sriov-net
-        multus:
-          networkName: sriov-net-crd
+#### How to expose SR-IOV VFs to KubeVirt
+In order to expose the SR-IOV VFs to KubeVirt, the SR-IOV capable NICs must
+first be configured. To do so, follow the [openstack guide on how to configure
+the compute nodes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_openstack_platform/7/html/networking_guide/sr-iov-support-for-virtual-networking#create-virtual-functions-on-the-compute-node).
+While that document mentions a different product, creating the Virtual
+Functions in the Kubernetes node is similar.
```

Comment:
Possibly not necessary.

Reply:
KubeVirt is not a product though!

I want to believe this pre-configuration is not necessary. The operator should enable VFs and iommu.

The documentation of the operator claims "Initialize the supported SR-IOV NIC types on selected nodes." as its feature.

Reply:
Removed this whole paragraph.